Packs/Core/ReleaseNotes/3_2_12.md

```diff
@@ -0,0 +1,45 @@
+
+#### Playbooks
+
+##### New: Msiexec execution of an executable from an uncommon remote location
+
+- New: This playbook addresses the following alerts:
+ 
+- Msiexec execution of an executable from an uncommon remote location with a specific port
+- Msiexec execution of an executable from an uncommon remote location without properties
+ 
+Playbook Stages:
+  
+Analysis: 
+ 
+- Check extracted domain's reputation
+- Check extracted domain's prevalence
+ 
+ 
+Investigation:
+ 
+- Check for suspicious related alerts: 
+    - Local Analysis Malware
+    - Mitre Techniques:
+        - T1140 - Deobfuscate/Decode Files or Information
+        - T1059 - Command and Scripting Interpreter 
+
+- Check for suspicious commandline arguments
+
+Containment:
```

Comment:
you are missing a lot of context in the descriptions. its not only the task you are running. when do you execute the kill causality? and more. please look at other playbook descriptions.